tests/src/buildApplication/generateFunctionsMap.test.ts

```diff
@@ -171,7 +171,7 @@ describe('generateFunctionsMap', async () => {
 			});
 		});
 
-		test('succeeds for prerendered favicon', async () => {
+		test('succeeds for prerendered favicon + json', async () => {
```

Comment:
```suggestion
		test('succeeds for prerendered favicon', async () => {
```

Reply:
derp i didn't change it when i copy pasted. oopsy

Reply:
it happens :slightly_smiling_face::+1: